backend/mora/common.py

```diff
@@ -50,6 +51,77 @@ def get(self, objid):
         pass
 
 
+class RequestType(enum.Enum):
+    CREATE = 0
+    EDIT = 1
+
+
+class Request(abc.ABC):
```

Comment:
To avoid confusing these for `requests.Request`, I'd suggest calling them `Handler` or something of that ilk.